modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -112,7 +121,69 @@ module.exports = {
         component: 'AposTiptapButton',
         label: 'Redo',
         icon: 'redo-icon'
+      },
+      alignLeft: {
+        component: 'AposTiptapButton',
+        label: 'Align Left',
+        icon: 'format-align-left-icon',
+        command: 'setTextAlign',
+        commandParameters: 'left',
+        isActive: { textAlign: 'left' }
+      },
+      alignCenter: {
+        component: 'AposTiptapButton',
+        label: 'Align Center',
+        icon: 'format-align-center-icon',
+        command: 'setTextAlign',
+        commandParameters: 'center',
+        isActive: { textAlign: 'center' }
+      },
+      alignRight: {
+        component: 'AposTiptapButton',
+        label: 'Align Right',
+        icon: 'format-align-right-icon',
+        command: 'setTextAlign',
+        commandParameters: 'right',
+        isActive: { textAlign: 'right' }
+      },
+      alignJustify: {
+        component: 'AposTiptapButton',
+        label: 'Align Justify',
+        icon: 'format-align-justify-icon',
+        command: 'setTextAlign',
+        commandParameters: 'justify',
+        isActive: { textAlign: 'justify' }
+      },
+      highlight: {
+        component: 'AposTiptapButton',
+        label: 'Mark',
+        icon: 'format-color-highlight-icon',
+        command: 'toggleHighlight'
       }
+    },
+    // Additional properties used in executing tiptap commands
+    // Will be mixed in automatically for developers
+    tiptapTextCommands: {
+      setNode: [ 'p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'pre' ],
+      toggleMark: [
+        'b', 'strong', 'code', 'mark', 'em', 'i',
+        'a', 's', 'del', 'strike', 'span', 'u'
+      ],
+      wrapIn: [ 'blockquote' ]
+    },
+    tiptapTypes: {
+      heading: [ 'h1', 'h2', 'h3', 'h4', 'h5', 'h6' ],
+      paragraph: [ 'p' ],
+      textStyle: [ 'span' ],
```

Comment:
What is this doing presently?

Reply:
This allows devs to pass in spans with classes to the RTE styles control. `textStyle` is the tiptap extension in charge of working with spans